Functions.Templates/Templates/SqlInputBinding-Python/__init__.py

```diff
@@ -1,9 +1,7 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
 import azure.functions as func
 import json
 
+# Visit https://aka.ms/sqlbindingsinput to learn how to use this input binding
```

Comment:
It would actually be nice to have a full doc comment explaining what this is with parameters and whatnot. See https://github.com/Azure/azure-functions-templates/pull/1204/files#diff-99f53da4f44d874e708c322ca153e20e6214725c2ea1561af46ab7331be4ebf6R3 for an example